RESTfulService/FSharp/FSharp/Program.fs

```diff
@@ -283,36 +283,36 @@ type UnitTest() =
 
     // API endpoints test.
     [<Test>]
-    member this.``API: GET /add``() =
+    let ``API: GET /add``() =
```

Comment:
This doesn't work on Mono:

```
/home/dima/github/RealtyShares/RealtySharesPublic/RESTfulService/FSharp/FSharp/Program.fs(5,5): Error FS0960: 'let' and 'do' bindings must come before member and interface definitions in type definitions (FS0960) (FSharp)
```

Is it just because my .NET runtime on Linux is outdated?